utilgo/starter/init.go

```diff
@@ -53,3 +55,12 @@ func GetCommandDelegatorServerClientConfig() string {
 	}
 	return address
 }
+
+func GetNoAuthClientConfig() string {
+	noAuthStatus := noAuthDefaultValue
+	if val := os.Getenv(noAuth); val != "" {
+		noAuthStatus = val
+	}
+	return noAuthStatus
```

Comment:
Just check the status manually here and return a boolean value. Also use lowercase true/false (I think theres another variable thats uses booleans, and uses lowercase).